app/js/containers/feedbackify_container.js

```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux'
+import Feedbackify from '../components/feedbackify/feedbackify'
+import { openFeedbackDialog } from '../actions/feedbackify'
+
+const mapStateToProps = (state) => {
+  // There are no props to map, but we need a function to call connect()!
+  return {}
```

Comment:
This should read the `email` from the state, so it can be provided to the Feedbackify container and to the click method that will hold the logic.